package.json

```diff
@@ -33,6 +33,7 @@
   "devDependencies": {
     "babel-core": "6.26.0",
     "babel-eslint": "8.0.2",
+    "babel-gettext-extractor": "git+https://github.com/muffinresearch/babel-gettext-extractor.git#0d39d3882bc846e7dcb6c9ff6463896c96920ce6",
```

Comment:
@muffinresearch can we at some point get whatever changed into upstream so that we don't have to rely on git-dependencies? I fear that's bad for packaging up the linter since users need to have `git` installed as well.